makejdk-any-platform.1

```diff
@@ -119,6 +119,9 @@ the one you are trying to build.
 .BR \-k ", " \-\-keep
 if using docker, keep the container after the build.
 .TP
+.BR \-C ", " \-\-make-args " " \fI<args>\fR
```

Comment:
I think there's a confusion clash here as `-C` is used for `--configure-args` in _config_init.sh_?

Reply:
copypasta - fixing